Makefile

```diff
@@ -70,7 +70,11 @@ export GOOS ?= $(TARGET_OS_LOCAL)
 export BINARY_EXT ?= $(BINARY_EXT_LOCAL)
 
 TEST_OUTPUT_FILE ?= test_output.json
-E2E_SH_TEST_TIMEOUT ?= 10m
+
+# Set the default timeout for tests to 10 minutes
```

Comment:
is not this same as `E2E_SH_TEST_TIMEOUT ?= 10m.

Reply:
Turns out, no. `?=` only sets a variable if it's not already set. In our case, it was set, but to `""`.